content/docs/deployment.md

```diff
@@ -208,6 +239,37 @@ Shards and replicas are some configuration values to take special attention to,
 index creation. [This article](https://qbox.io/blog/optimizing-elasticsearch-how-many-shards-per-index) goes into
 more information about choosing how many shards should be chosen for optimization.
 
+### Kafka
+Supported in Jaeger since 1.6.0
+Supported Kafka versions: 0.8+
+
+Currently, the Kafka storage backend implementation only supports writing data, this means you will need to use another one (with the multiple storage types feature) to be able to view the traces in the Query component.
```

Comment:
>Currently, the Kafka storage backend implementation only supports writing data

This is no longer true, but we don't yet have documentation for the `ingester`. 

>another one (with the multiple storage types feature) to be able to view the traces in the Query component.

I'd replace one with storage

Reply:
@vprithvi regarding the ingester, afaik it's not even released? It would probably be bundled as part of 1.7 so I can mention that instead of "in a later release"